wye/templates/reports/index.html

```diff
@@ -17,6 +17,8 @@ <h3 class="space-1q-top"> Report from {{date}}</h3>
                     <h3 class="space-1q-top"> Number of participants: {{no_of_participants}}</h3>
```

Comment:
I think these came on merging upstream/master in origin/master, because I have not added them explicitly. Should I remove them?